grasp_utils/robot_interface/README.md

```diff
@@ -0,0 +1,28 @@
+# robot_interface
+
+ROS2 package to use robot native interface
+
+## Install
+
+Install dependency **ur_modern_driver**:
+
+```shell
+git clone -b libur_modern_driver https://github.com/RoboticsYY/ur_modern_driver.git
+cd libur_modern_driver
+mkdir build && cd build
+cmake .. && sudo make install
+```
+
+Install **robot_interface**:
+
+```shell
+mkdir -p ~/ws_ros2/src && cd ~/ws_ros2/src
+git clone https://github.com/RoboticsYY/robot_interface.git
```

Comment:
can you check the accessibility of this repo?

Reply:
The content of this PR contains this repo. Before the PR is merged, I will continue to keep it as private. I will assign an access to you.